crm_request/models/request.py

```diff
@@ -49,7 +52,12 @@ def message_new(self, msg, custom_values=None):
         """
         if custom_values is None:
             custom_values = {}
-        custom_values.update({'description': msg.get('body')})
+        txt = msg.get('to')
+        alias_char = txt[txt.find('"') + 1:txt.find("@")]
```

Comment:
That could possibly fail if the `to` field contains another format or unexpected destination. In that case all operation will be aborted which could lead to losing information.
```suggestion
        alias_char = txt.split("@")[0].replace('"', '')
```